### ISSUE TITLE
rpi2 UX

### ISSUE THREAD

**Issue by kaihendry** [minio/minio#974]
I ran `go get -u github.com/minio/minio` on my alarm rpi2 system and it finishes with:

```
./main.go:110: app.HideVersion undefined (type *cli.App has no field or method HideVersion)
```

Not sure where the binary is.

```
   Static hostname: alarmpi
         Icon name: computer
        Machine ID: 84dadb5e3d8a4370ae51d251350c4943
           Boot ID: a3c880f3dbc34dff8eac4228d44e8d09
  Operating System: Arch Linux ARM
            Kernel: Linux 4.1.13-1-ARCH
      Architecture: arm
```

**Comment by harshavardhana** [comment]
@kaihendry Built binary is available here
- https://dl.minio.io:9000/updates/minio/2015/Nov/linux-arm/minio
- For all other platforms you can visit - http://minio.io/#minio

**Comment by harshavardhana** [comment]
Thanks for your bug report @kaihendry - there has been some changes in minio/cli, we forgot to update 'minio/minio'.  It should be fixed now #975
